Source/WebCore/platform/graphics/SourceBufferPrivateClient.h

```diff
@@ -69,6 +70,7 @@ class SourceBufferPrivateClient : public ThreadSafeRefCountedAndCanMakeThreadSaf
 
     struct InitializationSegment {
         MediaTime duration;
+        MediaPlayerVideoPlaybackConfiguration videoConfiguration;
```

Comment:
Can we call this `videoPlaybackConfiguration` or `playbackConfiguration` rather than `videoConfiguration`? It's a little too close to `videoTrackConfiguration`.